docs/source/support/releases.rst

```diff
@@ -0,0 +1,84 @@
+.. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Releases and Versioning
```

Comment:
This feels too detailed for external users, do we really want to document how exactly we select RCs, whether we do head-based development or not, …?

Reply:
I don't really say "how" we do it, just that we do it. I think it's nice to give some indication of what happens behind the curtain and this seems harmless to me, but I don't feel strongly about it. Maybe @cmblitz has some thoughts on this, too.

Reply:
My first reaction was to agree with @cocreature that was too detailed.  But I think you should check with other developers to see whether they appreciate this info or just find it extraneous.

Reply:
Maybe just move the process to the very end. versioning, cadence, release notes and roadmap seem clearly useful. The process less so, so it shouldn’t hide the useful things.

Reply:
done.